scripts/get_previous_timestamp.py

```diff
@@ -35,24 +41,38 @@ def get_workflow_runs(token: str, repo: str):
               "per_page": 100}
     url = f"https://api.github.com/repos/{repo}/actions/runs"
     r = requests.get(url, params)
-    runs = json.loads(r.text)
+    if r.status_code >= 500:
+        with open("patchwork_down.txt", "w") as f:
+            f.write(f"status code: {r.status_code}")
+        return None
+    runs = [run for run in json.loads(r.text) if run["name"] == workflow]
+    print(f"After filter have {len(runs)} to consider")
     return runs
 
 def get_run_timestamp(runs, run_id: str):
-    runs = [run for run in runs if run["name"] == "Patchworks"]
```

Comment:
Why remove this?
https://github.com/patrick-rivos/riscv-gnu-toolchain/commit/74362c59f40fca31545c8359698af2829fe739d7
Presumably we don't want to count the nightly-run as an actual run?

Reply:
nvm, filter is above